about.html

```diff
@@ -11,5 +11,20 @@
 </head>
 <body id="body">
     <script type="module" src="./about.js"></script>
+    <script type="module" src="./menu.js"></script>
 </body>
+<nav id="mobile-menu-page-id" class="mobile-menu-page-class">
+    <div id="mobile-menu-close-id" class="mobile-menu-close-class">
+        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+          <path stroke-linecap="round" stroke-linejoin="round" d="M6 18L18 6M6 6l12 12" />
+        </svg>
+    </div>
+    <ul id="moblie-menu-list-ul">
+        <li>
+            <a href="#ams" class='link'>
+                About
+            </a>
+        </li>
+    </ul>
+ </nav>
```

Comment:
- Hey, in the Mobile version of the About page mobile menu, the link should take the users to the homepage which is the index.html page. let's improve on this.